src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -166,17 +166,28 @@
     }
   }
 
-  function canPasteActivityDirectives(): boolean {
-    return plan !== null && hasCreatePermission && canPasteActivityDirectivesFromClipboard(plan);
+  function canPasteActivityDirectives(): Promise<number> {
+    return new Promise(resolve => {
+      getActivityDirectivesClipboardCount().then(result => {
+        if (plan !== null && hasCreatePermission && result > 0) {
```

Comment:
I think it's okay to use `hasCreatePermission` here, but our convention for permissions has been to disallow the user from being able to click on the option to paste and displayed a tooltip stating why they're unable to click. It just lets them know quicker why something isn't available to them.